src/client.cpp

```diff
@@ -52,6 +52,7 @@ std::string             g_szUserPath   = "";
 std::string             g_szClientPath = "";
 CHelper_libXBMC_addon  *XBMC           = NULL;
 CHelper_libXBMC_pvr    *PVR            = NULL;
+bool                   b_recordingPlayback = false;
```

Comment:
strange naming. Maybe this should read g_* to denote a global var?

Reply:
Sorry, should have been m_bRecordingPlayback.

It’s not actually global at all.

Reply:
I'm confused. AFAIR ee use m_ only for class member vars. But you are right with this var not being a global var. It's visible throughout the compilation unit only. Shouldn't it be 'static', then?

Reply:
Hmmm. Or just `bRecordingPlayback`?

Looking at several addons they appear to use either an `m_` or omit it in client.cpp.

No addon I can see uses static in this file.

Reply:
I will go with `m_`. Because when we go to C++ this will be member variable.